### ISSUE TITLE
VS bundling & minification doesn't produce output files

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#30]
### Brief bug description

The stock Visual Studio bundling & minification infrastructure doesn't produce any results with the bundleconfig.json file.

### Repro steps

1. Build the solution.
1. Observe the /wwwroot/css/ and /wwwroot/js/ output folders.

### Expected behavior

The site.min.css, site.min.js, and materialize.min.js files should be built.

### Test environment

 - Windows 10 Pro 1909 x64 en-US
 - Visual Studio 2019 16.6.0

### Additional context

The bundleconfig.json file fully adheres to the [documentation](https://docs.microsoft.com/en-us/aspnet/core/client-side/bundling-and-minification?view=aspnetcore-3.1).

### Screenshots

Add links to screenshots, if you deem it appropriate.

### Issue expands upon

This issue expands upon another issue(s): #29 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
